google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionOptions.java

```diff
@@ -959,6 +969,11 @@ public TransportChannelProvider getChannelProvider() {
     }
   }
 
+  /** The creator role to use for the connection. */
```

Comment:
I don't think users will understand what 'creator role' means. This seems like an internal name (I (think I) understand that it means `the role that is used to create a session`, but that is not clear to a user.)

I would suggest: `The database role that is used for this connection. Assigning a role to a connection can be used to for example restrict the access of a connection to a specific set of tables.`